Remotion/Web/Core/UI/Controls/Command.cs

```diff
@@ -970,6 +970,7 @@ public virtual WxeFunctionCommandInfo WxeFunctionCommand
     [DesignerSerializationVisibility (DesignerSerializationVisibility.Hidden)]
     [Browsable (false)]
     [EditorBrowsable (EditorBrowsableState.Advanced)]
+    [AllowNull]
```

Comment:
Okay, (see also the other comments), now I remember AllowNull. Can we drop this and just keep the null-coalesce. It probalby makes things less ambiguous going forward.

Reply:
Why would we drop it and keep dead code? The fallback value would stay there for years. If AllowNull should be removed, then the fallback value too imo.

Reply:
the problem ist, that NRT isn't something that's enforcable right now for projects. It's just a usage hint as an opt-in. (same reason we can't drop the arg-null checks). I just figure, the AllowNull isn't helping so we can drop it while the fallback makes our code safe for our own usage.

=> I would prefer to mark the "here we could pass null as well" information as obsolete, but that's not possible. So, dropping the attribute accomplishes the same effect, to me. TBH, if that's not a good reason, we can keep the attribute. It's a gut feeling thing about how the api is used, that's all.

Reply:
When dropping the nullability and adding an argument check, the contract is enforced for projects - would this be the way to go?

Reply:
this would break the project if they do not use NRT and not notice that null is not recommended here. That's the problem with weak constraints. We can just fallback during the setter execution to a not=null value. The only reason to not officially allow null is that it's not really helpful to pass null. So, we keep the attribute and just shelve this?

Reply:
Isn't this what breaking changes are for?

I don't have a strong opinion on what to do here, I just ask the questions to have the full understanding for future reference.

Reply:
Yeah, excatly. I just don't want to have a breaking change for this because it's not helpful to anybody. It's mainly based on the consideration of what's helpful for the application developer and doesn't introduce bugs. So, if we make this a breaking change, it would only be discovered by reading the docs and then looking up the usages. That doesn't happen. Thus, breaking changes that don't express themselves as a compiler error need to be carefully evaluated and published very visibly. So, this particualr change just isn't worth it (nobody is going to benefit from us locking down the setter). 

The soft hint (not having the AllowNull attribute) on the other hand might give someone a compiler error after they switch, they will notice and clean it up. But if they down't nothing breaks, more satisfaction all round.